alerts/system_alerts.libsonnet

```diff
@@ -59,6 +59,151 @@
               message: "Kubernetes API server client '{{ $labels.job }}/{{ $labels.instance }}' is experiencing {{ printf \"%0.0f\" $value }} errors / sec.'",
             },
           },
+          {
+            alert: 'KubeletDown',
+            expr: |||
+              absent(up{%(kubeletSelector)s} == 1)
+            ||| % $._config,
+            'for': '15m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'Kubelet could not be scraped or all kubelets disappeared from Prometheus target discovery.',
+            },
+          },
+          {
+            alert: 'KubeletTooManyPods',
+            expr: |||
+              kubelet_running_pod_count{%(kubeletSelector)s} > 100
```

Comment:
I suspect this one may need to have a configurable threshold, wdyt @tomwilkie ?

Reply:
I can't hurt, and I seem to recall you use a different limit for openshift?